cla_backend/apps/call_centre/sql/ExportStandardTables.sql

```diff
@@ -0,0 +1,30 @@
+CREATE OR REPLACE FUNCTION cla_db_to_csv(dt_from TEXT, dt_to TEXT, path TEXT) RETURNS void AS $$
```

Comment:
oh if it runs every time then its fine; prefer that to a migration

keep it like it is